src/SamplesApp/UITests.Shared/Lottie/LottieProgressPage.xaml.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Uno.UI.Samples.Controls;
+using System.Threading.Tasks;
+
+namespace UITests.Lottie
+{
+	[Sample("Lottie", Name="Lottie Progress")]
+	public sealed partial class LottieProgressPage : Page
+	{
+		private const double ClickedSegmentFromProgress = 0.69;
+
+		public LottieProgressPage()
+		{
+			this.InitializeComponent();
+
+			// Set Lottie Source.
+			var _ = InitializeJsonSource();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable '_'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5436651)